internal/runners/packages/packages.go

```diff
@@ -119,27 +120,14 @@ func executePackageOperation(pj *project.Project, cfg configurable, out output.O
 		commitID = parentCommitID
 	}
 
-	// Create runtime
-	rtMessages := runbits.NewRuntimeMessageHandler(out)
-	rtMessages.SetRequirement(name, ns)
-	rt, err := runtime.NewRuntime(pj.Source().Path(), cfg.CachePath(), commitID, pj.Owner(), pj.Name(), rtMessages)
-	if err != nil {
-		return locale.WrapError(err, "err_packages_update_runtime_init", "Could not initialize runtime.")
-	}
-
-	if !orderChanged && rt.IsCachedRuntime() {
-		out.Print(locale.Tl("pkg_already_uptodate", "Requested dependencies are already configured and installed."))
-		return nil
+	// refresh runtime
+	target := refreshTarget{
```

Comment:
Can we name this `refreshRequirement`. The terminology used throughout is "requirement".